ui/component/claimTilesDiscover/view.jsx

```diff
@@ -176,17 +176,23 @@ function ClaimTilesDiscover(props: Props) {
   }, [doClaimSearch, shouldPerformSearch, optionsStringified]);
 
   React.useEffect(() => {
+    refreshBuffer();
+  }, [finalUris, injectedItem, lastVisibleIndex, pageSize]);
+
+  function refreshBuffer() {
     finalUris.forEach((uri, index) => {
       if (uri) {
         const inj = getInjectedItem(index);
         if (inj) {
           if (uriBuffer.indexOf(index) === -1) {
-            setUriBuffer([index]);
+            let newUriBUffer = uriBuffer;
+            newUriBUffer.push(index);
+            setUriBuffer(newUriBUffer);
           }
         }
       }
     });
-  }, [finalUris, injectedItem, lastVisibleIndex, pageSize]);
+  }
```

Comment:
At the time of writing, Featured is missing one tile when compared to the production.

Looking at the `claim_search` results, production looks correct.  I should have 113 tiles, so 1 should be left hanging.

<img width="1984" alt="image" src="https://user-images.githubusercontent.com/64950861/170916666-444c67f2-7395-48b1-97bc-9124b074109e.png">

Reply:
There are multiple functions at work to keep the grid even. Single tile lines get avoided on purpose unless the single column mode is active.

Reply:
Hm ... not clear what this means.
In Channel Page, would that cause a missing tile?

Reply:
Nope, channel pages should always be complete if you scroll to the bottom. Only the homepage and the categories get "cut" in order to keep the grid even. Unless there's something else happening here.